utils/gui.py

```diff
@@ -53,22 +53,31 @@ def windowRefocus(name):
 
 class Gui:
```

Comment:
I dont think this class is used anymore

Reply:
its not

Reply:
I copied the updated one in merge conflict

Reply:
In that case, we should replace all usage of Gui with your new window objects and completely eradicate the unused code.